candidates/index_config/works-candidate.json

```diff
@@ -805,6 +821,10 @@
               "base": {
                 "type": "text",
                 "analyzer": "base"
+              },
+              "normalized_keyword": {
+                "type": "text",
+                "analyzer": "normalized_keyword"
```

Comment:
I think this field is only used in prefix queries.  This could be a good candidate for [index_prefixes](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-prefixes.html)

Reply:
Oooh! Yes.

Reply:
Hmmm... or maybe no? A `prefix` query doesn't (it turns out) do any analysis, which is why we use `match_phrase_prefix` here to get the normalization benefits. I can't find any evidence that `match_phrase_prefix` makes any use of `index_prefixes`, but I also can't find any evidence that it doesn't...